### DISCUSSION TITLE
21-06-08 일일회고

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#18] [회고]
6명 모두 코쿼출석! 🎯

**Comment by Jiwon-JJW** [comment]
깃허브 Oauth 백엔드 구현은 완료!! 내일은 빠르게 서버 구현 하겠습니다 ㅎㅎ

**Comment by GleamingStar** [comment]
리스트 화면 조금 구현했네요!
내일은 이슈 목록을 목데이터로 구현해볼 생각입니다

**Comment by eeeesong** [comment]
잭슨과 함께 로그인 창 UI를 만들었습니다!
내일은 OAuth를 적용해보지 않을까 싶네요 :)
다들 오프라인에서 만나서 반가웠어요!

**Comment by JacksonPk** [comment]
늦게 적어서 죄송합니다. 어제 막국수 너무 맛있었어요!
또한 로그인 UI를 코딩온리로 구현해서 좋았습니다. 다들 오늘도 화이팅입니다~

**Comment by reminderclock** [comment]
회고가 많이 늦었네요ㅠㅠ, 어제 로그인 헤더 ui 정도만 구현하고 라우터 설계도 조금 고민해봤습니다. 어제 정말 반가웠어요:)
